scss/bootstrap/overrides/_variables.scss

```diff
@@ -352,10 +347,21 @@ $state-info-text:                $brand-info !default;
 $state-warning-text:             $brand-warning !default;
 $state-danger-text:              $brand-danger !default;
 
-$alert-success-border:           $brand-success !default;
+$alert-success-border:           $btn-success-bg !default;
 $state-info-border:              $brand-info !default;
-$alert-warning-border:           $brand-warning !default;
+$alert-warning-border:           $btn-warning-bg !default;
 $alert-danger-border:            $brand-danger !default;
+
+$alert-success-bg:              lighten($btn-success-bg, 30%) !default;
```

Comment:
are these custom variables or are you overriding existing ones?

Reply:
This is overriding of existing variables.